source/dub/commandline.d

```diff
@@ -383,16 +650,90 @@ class CommandArgs {
 		m_args = null;
 	}
 
-	/** Returns the list of unprocessed arguments and calls `dropAllArgs`.
+	/** Returns the list of unprocessed arguments, ignoring the app arguments,
+	and calls `dropAllArgs`.
```

Comment:
but does it call dropAllArgs? While not really likely, it's possible someone inherits from CommandArgs and expects dropAllArgs to be called (for whatever reason) because of this documentation.